CoroutineTemplate/domain/src/main/java/co/nimblehq/coroutine/model/User.kt

```diff
@@ -0,0 +1,24 @@
+package co.nimblehq.coroutine.model
+
+data class User(
+    val id: Int?,
+    val name: String,
+    val username: String,
+    val email: String?,
```

Comment:
seems like we put the email fallback as empty and never null

Reply:
@sleepylee Updated